surah/src/main/res/layout/row_aya.xml

```diff
@@ -95,7 +96,7 @@
         android:textColor="@color/green_dark"
         android:textSize="18dp"
         tools:ignore="RtlHardcoded,SpUsage"
-        tools:text="بسم الله الرحمن الرحیم" />
+        tools:text="بسم الله الرحمن الرحیم"/>
```

Comment:
we have it in the `settings/strings.xml`

Reply:
it's `tools` so does not really matter